homeassistant/components/onvif/__init__.py

```diff
@@ -35,29 +25,6 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
This can be removed completely. The Requirement to keep setup was removed some time ago.

Reply:
I thought so, but one of the tests failed when I tried to remove it. Will have to go back and see why.

Reply:
Sweet!